packages/BUILD.bazel

```diff
@@ -34,6 +34,7 @@ filegroup(
       "//packages/kbn-optimizer:build",
       "//packages/kbn-plugin-generator:build",
       "//packages/kbn-plugin-helpers:build",
+      "//packages/kbn-pr-utils:build",
```

Comment:
There's one more reference in monorepo-packages.asciidoc.  I'm not sure that it's relevant anymore, but thought I'd note.

Reply:
One more reference?

Reply:
There's a list of bazel packages in that file.  I'm not concerned here, resolving.